fre/analysis/tests/test_subtools.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+import pytest
+from tempfile import TemporaryDirectory
+
+from analysis_scripts import find_plugins, available_plugins, UnknownPluginError
+from fre.analysis.subtools import install_analysis_package, run_analysis
+
+
+def make_experiment_yaml(path, name, whitespace="  "):
```

Comment:
i haven't dug into every last thing about `pytest` yet, but I would double check to make sure this function gets used the way you think its getting used and isn't being collected as a test.

there's definitely room in `pytest`'s framework for canon-izing these sorts of set-up routines. i'll get there someday if someone else doesn't first, but for now, i superstitiously start every `pytest` with `test_`